### PR TITLE
Properly policy submissions

### PR DESCRIPTION
**Pull request by kurtisnelson**
Viewing student submissions as an instructor was broken...so logically I setup Pundit and started writing policies. Also tests!

### CODE REVIEW

File: app/assets/javascripts/components/grade_submission_form.js.jsx [hunk 1]
```diff
@@ -16,7 +16,6 @@ var Nav = React.createClass({
     render: function () {
         return (
             <div>
-                <Link url='/instructor/dashboard' text="Return to Dashboard"/>
```
Comment (jah2488) [inline app/assets/javascripts/components/grade_submission_form.js.jsx]: Why remove this link?

**Comment by jah2488** [comment]
Out of curiosity, what was broken about viewing the submissions as an instructor?

**Comment by kurtisnelson** [comment]
It was redirecting to login.

File: app/assets/javascripts/components/grade_submission_form.js.jsx (hunk 1 above)
Comment (kurtisnelson) [inline app/assets/javascripts/components/grade_submission_form.js.jsx]: It's broken. I tried to fix it by passing it in as a variable but my react skills are bad.

**Pull request merged**